deploy/team-api-update.js

```diff
@@ -21,11 +34,11 @@ function spawn(action_description, path, args) {
   });
 }
 
-function bundleInstall() {
+TeamApiUpdater.prototype.bundleInstall = function bundleInstall() {
```

Comment:
Something @shawnbot told me a while back that using names on both sides helps with stack traces. I also just read about this in https://docs.npmjs.com/misc/coding-style#functions.